### PR TITLE
Remove test_that_searching_for_cool...

### PR DESCRIPTION
**Pull request by stephendonner**
Fixes https://github.com/mozilla/Addon-Tests/issues/781

### CODE REVIEW

File: tests/desktop/test_search.py [hunk 1]
```diff
@@ -92,26 +92,6 @@ def test_that_searching_for_firebug_returns_firebug_as_first_result(self, base_u
...
-        search_page = home_page.search_for(search_term)
-        assert not search_page.is_no_results_present
-
-        for i in range(0, len(search_page.results)):
-            try:
-                assert search_term in search_page.results[i].text.lower()
-            except:
-                devs_comments = ''
-                details_page = search_page.results[i].click_result()
-                if details_page.is_devs_comments_section_present:
```
Comment (davehunt) [inline tests/desktop/test_search.py]: I think this is the only place we use `is_devs_comments_section_present` so let's remove that too.

File: tests/desktop/test_search.py [hunk 2]
```diff
@@ -92,26 +92,6 @@ def test_that_searching_for_firebug_returns_firebug_as_first_result(self, base_u
...
-                    details_page.expand_devs_comments()
-                    devs_comments = details_page.devs_comments_message
-                search_range = details_page.description + devs_comments
-                assert search_term in search_range.lower()
-                details_page.return_to_previous_page()
```
Comment (davehunt) [inline tests/desktop/test_search.py]: As above, this appears to be the only place this is used, and I'd rather we used `selenium` directly for such actions anyway.

**Comment by davehunt** [comment]
r+ with comments

**Comment by stephendonner** [comment]
I removed too much in my most-recent commit/branch push (obviously without testing, like a dunce), so I'll fix that and re-push with only the removal of is_devs_comments_section_present.

Afterwards, I'll seek re-review; thx!

**Comment by stephendonner** [comment]
Ad-hoc (passing) job against staging, here: https://pastebin.mozilla.org/8875865

File: tests/desktop/test_search.py (hunk 2 above)
Comment (davehunt) [inline tests/desktop/test_search.py]: I still think we could remove this from page.py as it's not used anywhere else afaict.

**Comment by davehunt** [comment]
Looks good, just a nit about an unused method in page.py that could now be removed.

**Pull request merged**